sentence_transformers/SentenceTransformer.py

```diff
@@ -1119,7 +1119,7 @@ def encode(
     def forward(self, input: dict[str, Tensor], **kwargs) -> dict[str, Tensor]:
         for module_name, module in self.named_children():
             module_kwargs = {}
-            if isinstance(module, Router):
+            if isinstance(module, Router) or isinstance(module, Transformer):
```

Comment:
I'm not entirely sure how safe this change is but it seemed like the easiest way to make sure that the SentenceTransformer class passes the `output_attentions` flag to the `Transformer` model. Open to suggestions if this can lead to bugs.